### DISCUSSION TITLE
Player Level feature

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#668] [Ideas]
Points could be used to determine the level of the player (similiar feature is used on several gaming social networks like Steam for example). Additionaly reaching the level milestone (every 5 or 10 levels for example) the player could recieve seperate site award.

**Comment by kdecks** [comment]
I don't see the point.

**Comment by julkasalsa** [comment]
The point is about the fact that most of gaming systems have similiar structure, and it's acclaimed by players. This is more of decorative feature.

**Comment by Jamiras** [comment]
Having a level-up system measures personal growth, whereas the existing global leaderboard encourages competition between players, possibly leading to cheating that wouldn't otherwise exist as users try to climb the rank ladder.

It's possible for both to exist, but it would probably make more sense to eliminate the ranking system entirely, or only allow ranking compared to friends (which level would probably be sufficient to cover).

**Comment by kdecks** [comment]
I'm convinced now of the merit of an cosmetic XP system as a positive feedback system for the players.

Also, I'm for removing the global rank, leaving an emphasis on personal achievement and friend comparison and competition. There are too many cheaters at the top and most of the time it's unverifiable or prohibitively difficult to determine who with certainty. As it stands te global ranking rewards cheating and de-incentives honest play.

It makes things more casual, to a degree; is that a bad thing?
